vllm/spec_decode/spec_decode_worker.py

```diff
@@ -182,9 +182,12 @@ def create_worker(
                         draft_worker_kwargs[
                             "model_runner_cls"] = TP1DraftModelRunner
                 else:
-                    if draft_model_config.hf_config.model_type == "eagle":
+                    if draft_model_config.hf_config.model_type in [
+                            "eagle", "deepseek_mtp"
+                    ]:
```

Comment:
nit: Better to use tuple as it is immutable and should be (slightly) faster.
```suggestion
                    if draft_model_config.hf_config.model_type in (
                            "eagle", "deepseek_mtp"
                    ):
```